pkg/check/chunkrepair/chunkrepair.go

```diff
@@ -229,13 +229,16 @@ func uploadAndPinChunkToNode(ctx context.Context, node *bee.Client, chunk *bee.C
 	if err != nil {
 		return err
 	}
-	pinned, err := node.PinChunk(ctx, chunk.Address())
+
+	err = node.PinChunk(ctx, chunk.Address())
```

Comment:
`return node.PinChunk`

Reply:
I have left this as it was, because I was not sure if it would break existing code, but we will see it if it does I guess (probably should have had separate PR for this change).